addons/web/static/src/core/navigable_popover/components/dropdown.js

```diff
@@ -0,0 +1,201 @@
+                {{ state.isOpen ? 'o_dropdown--open' : ''}}
+                {{ hasParent ? 'o_dropdown--has_parent' : ''}}
+                {{ props.slots.toggler.class || props.togglerClass || ''}}
+                {{ !props.showCaret ? 'o_dropdown_no_caret' : '' }}
+            "
+            t-att-class="hasParent ? 'o_navigable_item' : ''"
+            t-att-disabled="props.disabled"
+            tabindex="0"
+        >
+            <t t-slot="toggler"/>
+            <i
+                t-if="props.showCaret"
+                class="ms-3 o_dropdown_caret fa fa-caret-down"
+                aria-hidden="true">
+            </i>
+        </button>
+        <Menu
+            isOpen="state.isOpen"
+            close.bind="close"
+            targetRef="togglerRef"
+            containerClass="'d-flex flex-column ' + props.containerClass"
+            options="options"
+            onOpened="() => this.props.onOpened and this.props.onOpened()"
+        >
+            <t t-slot="default"/>
+        </Menu>
+    `;
+    static components = { Menu };
+    static props = {
+        togglerClass: { type: String, optional: true },
+        containerClass: { type: String, optional: true },
+
+        disabled: { type: Boolean, optional: true },
+        showCaret: { type: Boolean, optional: true },
+
+        // Enables group system
+        autoOpen: { type: Boolean, optional: true },
+
+        // Custom Toggler
+        togglerRef: { type: Object, optional: true },
+
+        position: { type: String, optional: true },
+        onOpened: { type: Function, optional: true },
+        options: { type: Object, optional: true },
+        slots: { Object },
+    };
+    static defaultProps = {
+        togglerClass: "btn btn-secondary",
+        containerClass: "d-flex flex-column",
+        showCaret: true,
+        autoOpen: false,
+        options: {
+            navigation: {},
+            popover: {},
+        },
+    };
+
+    setup() {
+        this.togglerRef = this.props.togglerRef || useRef("togglerRef");
+        this.hasParent = useMenuHasParent();
+
+        this.state = useState({ isOpen: false });
+
+        this.options = {
+            navigation: this.props.options.navigation,
+            popover: {
+                ...this.props.options.popover,
+                position: this.props.position || (this.hasParent ? "right-start" : "bottom-start"),
+            },
+        };
+
+        if (this.props.autoOpen) {
+            useGroups(this.togglerRef, this.state);
+        }
+
+        if (this.props.togglerRef) {
+            useRefListener(this.props.togglerRef, "click", () => this.toggle());
+            useRefListener(this.props.togglerRef, "mouseover", () => this.hasParent && this.open());
+            useRefListener(this.props.togglerRef, "keypress", (ev) => {
+                if (document.activeElement === ev.target && ev.key === "Enter") {
+                    this.open();
+                }
+            });
```

Comment:
maybe not needed ?
I mean, a `<button/>` triggers a click event through the press of enter key by default (I think, did not check).
If a component would like to handle the enter keypress, it can do it itself.